### PR TITLE
Make progress bars stack vertically

### LINKED ISSUE #10128: Progress bar bugs
If you strip someone prone, the bar is rotated 90 degrees, but isn't above their head.

If you strip multiple items, multiple bars appear layered on top of one another, and each moves to the top of the pile each time the bar updates. I don't think there'd be an easy fix for this given the lack of screen space available on a tile (though maybe they could stack upwards, onto other tiles?), but I thought it was worth mentioning anyway.

Deconstructing broken turrets with a crowbar doesn't show a progress bar (probably a request, not a bug)

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #10128

:cl: Cyberboss
fix: Progress bars will now stack vertically instead of on top of each other
fix: Progress bars will no longer be affected by lighting
/:cl:
untitled

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Can you also make these be like, hud layer? so you can see them even in complete darkness

**Comment by MrPerson** [comment]
This won't put them above the lights because plane takes priority over layer.

File: code/datums/progressbar.dm [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
 /datum/progressbar/New(mob/User, goal_number, atom/target)
 	. = ..()
 	if (!istype(target))
 		EXCEPTION("Invalid target given")
 	if (goal_number)
 		goal = goal_number
-	bar = image('icons/effects/progessbar.dmi', target, "prog_bar_0")
+	bar = image('icons/effects/progessbar.dmi', target, "prog_bar_0", HUD_LAYER)
 	bar.appearance_flags = APPEARANCE_UI_IGNORE_ALPHA
-	bar.pixel_y = 32
```
Comment (MrPerson) [inline code/datums/progressbar.dm]: bar.plane = HUD_PLANE

**Comment by Cyberboss** [comment]
In addition to the hud layer or without it?

**Comment by MrPerson** [comment]
Doesn't matter, render order is plane then layer within the plane.

**Pull request merged**